core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -58,6 +58,9 @@ contract PricelessPositionManager is FeePayer {
     uint public expirationTimestamp;
     // Time that has to elapse for a withdrawal request to be considered passed, if no liquidations occur.
     uint public withdrawalLiveness;
+    // Time that has to elapse for the contract to be siphoned. This occures if a long period (>6 months) has
+    // occured post settlement and there is remaining unclaimed collateral in the contract.
```

Comment:
nit: should be post "expiration" not post "settlement" right? Just want to make sure I'm understanding properly

Reply:
in this context is settlement not the same expiration?